pkg/parser/parser.go

```diff
@@ -49,13 +51,15 @@ var supportedEcosystems map[string]bool = map[string]bool{
 	models.EcosystemCyDxSBOM:      true,
 	models.EcosystemSpdxSBOM:      true,
 	models.EcosystemGitHubActions: true,
+	models.EcoSystemTerraform:     true,
 }
 
 // TODO: Migrate these to graph parser
 var customExperimentalParsers map[string]lockfile.PackageDetailsParser = map[string]lockfile.PackageDetailsParser{
 	customParserTypePyWheel: parsePythonWheelDist,
 	customParserSpdxSBOM:    spdx.Parse,
 	customParserTypeSetupPy: py.ParseSetuppy,
+	customParserTerraform:   terraform.ParseTerraformLockfile,
```

Comment:
I think your parser can be reliable and need not be in `experimental` list. You can move your parser to `graph` parsers which is the preferred model for implementing parser going forward. You can simply move the definition to

https://github.com/safedep/vet/blob/main/pkg/parser/parser.go#L85

Reply:
The reason I moved this to experimental was because I thought that it needed to be tested maybe in prod a few times and then we will move to graph , will make this change